src/main/scala/com/codacy/analysis/cli/analysis/Status.scala

```diff
@@ -0,0 +1,36 @@
+package com.codacy.analysis.cli.analysis
+
+import com.codacy.analysis.cli.command.analyse.AnalyseExecutor.ExecutorResult
+
+object Status {
+
+  object ExitCodes {
+    val success = 0
+    val failedAnalysis = 1
+    val partiallyFailedAnalysis = 2
+    val maxAllowedIssuesExceeded = 101
+  }
+}
+
+class Status(maxAllowedIssues: Int, failIfIncomplete: Boolean = false) {
```

Comment:
Highly dislike this class name. Maybe SuccessCriteria? Or something else that points to this being an optional failure threshold / configuration / settings.